packages/flutter_test/lib/src/window.dart

```diff
@@ -330,6 +330,37 @@ class TestWindow implements Window {
     _window.onPlatformMessage = callback;
   }
 
+  /// Web specific members which must be present for compilation.
```

Comment:
cc @yjbanov We'll need to move or remove this methods due to the `implements` clause.

Reply:
Yep, filed as https://github.com/flutter/flutter/issues/33493